manifests/init.pp

```diff
@@ -317,8 +317,8 @@
   Integer[0] $tcp_backlog                                        = 511,
   Integer[0] $tcp_keepalive                                      = 0,
   Integer[0] $timeout                                            = 0,
-  Stdlib::Absolutepath $unixsocket                               = '/var/run/redis/redis.sock',
-  Stdlib::Filemode $unixsocketperm                               = '0755',
+  Variant[Stdlib::Absolutepath, Enum['']] $unixsocket            = '/var/run/redis/redis.sock',
+  Variant[Stdlib::Filemode, Enum['']] $unixsocketperm            = '0755',
```

Comment:
Is it really needed ?

Reply:
Previously the template allowed it.

Reply:
I see only one use case , let the default umask manage the perm of the file.
Bah ... not sure it is useful, but why not.